benchmarks/admin.py

```diff
@@ -18,7 +18,12 @@ class ResultAdmin(admin.ModelAdmin):
 @admin.register(TestJob)
 class TestJobAdmin(admin.ModelAdmin):
     list_display = ('id', 'url', 'status', 'completed', 'created_at')
+    actions = ['force_fetch_results']
 
+    def force_fetch_results(self, request, queryset):
+        for testjob in queryset.all():
+            set_testjob_results.delay(testjob)
+    force_fetch_results.short_description = "Force fetch results"
```

Comment:
Probably my fault. This is only visible in admin interface. It might be a separate patch so it doesn't 'pollute' the fix for the bug. The reason for including this code is to provide ability to 're-fetch' results that art-reports is missing. So in a way it also 'fixes' the bug (or rather the results of the bug).